_data/tools/editors.yml

```
@@ -0,0 +1,20 @@
+- name: Xcode
+  description: Xcode is a complete IDE enabling you to develop, test, debug, and distribute apps for all Apple platforms on macOS.
+  link: https://developer.apple.com/xcode
+  guide: https://developer.apple.com/xcode
+- name: Visual Studio Code
+  description: Visual Studio Code is a cross-platform and extensible editor that supports Swift through the Swift extension, which provides intelligent editor functionality as well as debugging and test support.
+  link: https://code.visualstudio.com
+  guide: /documentation/articles/getting-started-with-vscode-swift.html
+- name: Emacs
+  description: Emacs is an extensible cross-platform text editor. Swift editing support can be enabled through community provided packages.
+  link: https://www.gnu.org/software/emacs/
+  guide: /documentation/articles/zero-to-swift-emacs.html
+- name: Neovim
+  description: Neovim is an extensible cross-platform text editor. Swift editing support can be enabled through your pick of community provided plugins.
+  link: https://neovim.io/
+  guide: /documentation/articles/zero-to-swift-nvim.html
+- name: Other Editors
+  description: Any editor that supports the Language Server Protocol (LSP) can use SourceKit-LSP to provide intelligent editor functionality for Swift.
+  link: https://github.com/apple/sourcekit-lsp/tree/main/Editors#other-editors
+  guide: https://github.com/apple/sourcekit-lsp/tree/main/Editors#other-editors
```

I’m moving this file in https://github.com/apple/sourcekit-lsp/pull/1415. In anticipation of that, could you adjust the link to

```suggestion
  link: https://github.com/apple/sourcekit-lsp/tree/main/Documentation/Editor%20Integration.md
  guide: https://github.com/apple/sourcekit-lsp/tree/main/Documentation/Editor%20Integration.md
```

---

done!